posthog/models/organization_domain.py

```diff
@@ -161,11 +160,6 @@ def attempt_verification(self) -> Tuple["OrganizationDomain", bool]:
         """
         Performs a DNS verification for a specific domain.
         """
-
-        if not is_cloud():
```

Comment:
Is this intentional?

Reply:
yep, people won't be allowed to create OrganizationDomains on self-hosted now, so we don't need this check.